docs/options.md

```diff
@@ -304,3 +304,7 @@ with info about your project (name and website) so we can add an entry for you.
 1. Embedded Proto
    * Website: https://EmbeddedProto.com
    * Extension: 1141
+1. Protoc-gen-fieldmask
```

Comment:
Could you add a blank line above this one?

Reply:
done